diff --git a/texts/es/DW_es.tok b/texts/es/DW_es.tok
index 057e16d..ba07f17 100644
--- a/texts/es/DW_es.tok
+++ b/texts/es/DW_es.tok
@@ -420,8 +420,7 @@ América Latina
 Investigando para el gran público 
 Con un sobresaliente proyecto , la Academia de Deutsche Welle apoya a periodistas dedicados a la investigación en América Latina . 
 El centro de la iniciativa es el weblog re-visto . 
-Para Steffen Leidel , una cosa es especialmente importante : 
-" re-visto es mucho más que un blog . " 
+Para Steffen Leidel , una cosa es especialmente importante : " re-visto es mucho más que un blog . "
 Leidel es coordinador de un proyecto internacional de medios que tiene como objetivo promover el periodismo de investigación en América Latina . 
 " Creamos esta plataforma para impulsar una red de colegas de distintos países , además de ofrecer un foro para la teoría y la práctica del periodismo de investigación " , dice Leidel . 
 Los comienzos de re-visto se remontan a talleres realizados en Bolivia y Perú . 
@@ -484,8 +483,7 @@ A continuación discutimos acerca de la situación de la televisión infantil en
 El resultado : " Nos percatamos de que en Irak no existen series televisivas para niños . " 
 Por eso , uno de los objetivos del taller consistió en identificar e investigar temas infantiles , para darles el formato adecuado y presentarlos en programas de televisión dirigidos a la niñez . 
 " Esto " , dice Daniela Leese , " era algo prácticamente desconocido para los participantes . " 
-También para Aso Tareq , camarógrafo de Bagdad : 
-" Apenas había comenzado el taller cuando me di cuenta de que podía tirar por la borda todo lo que anteriormente sabía . 
+También para Aso Tareq , camarógrafo de Bagdad : " Apenas había comenzado el taller cuando me di cuenta de que podía tirar por la borda todo lo que anteriormente sabía .
 Para mí , el curso fue como comenzar desde el principio . " 
 A la teoría siguió la práctica . 
 En la fase de producción , que duró una semana , surgieron seis cápsulas para un programa piloto que serviría como ejemplo : perfiles breves , reportajes y artículos de servicio , hechos a la medida de un público conformado por niños y niñas de entre cinco y once años de edad . 
@@ -840,8 +838,7 @@ Deutsche Welle
 La emisora alemana para el exterior 
 " Debe presentar a Alemania como una nación cultural de labrado europeo y como un Estado de derecho concebido en libertad " . 
 Debe " dar foro a visiones alemanas y a otras visiones en lo que se refiere a temas relevantes tanto en Europa como en otros continentes , con la meta de promover el entendimiento y el diálogo entre las culturas y los pueblos " . 
-Con ello debe contribuir además a la promoción de la lengua alemán . 
-Así reza su mandato legal . 
+Con ello debe contribuir además a la promoción de la lengua alemán .  Así reza su mandato legal .
 La Deutsche Welle cumple con dicho mandato con su oferta periodística en televisión , radio e internet ( www.dw-world.de ) : televisión en alemán , inglés , árabe y español ; radio e internet en 30 idiomas , del amárico hasta el urdu , pasando por el indonesio y el suahili . 
 Se apoya para ello en un equipo de cerca de 1.500 trabajadores fijos así como de cientos de colaboradores libres de 60 países . 
 La Deutsche Welle se dirige a personas en todo el mundo que se interesan por Alemania y Europa , en especial a multiplicadores y tomadores de decisiones ; y en Estados autoritarios , a quienes luchan de manera activa en favor de la democracia , el progreso y los derechos libertarios . 
diff --git a/texts/quz/DW_quz.txt b/texts/quz/DW_quz.txt
index 876e273..094e14b 100644
--- a/texts/quz/DW_quz.txt
+++ b/texts/quz/DW_quz.txt
@@ -1,5 +1,4 @@
-﻿Academia 'Deutsche Wellep' 2009 wata  hatun willaykuriynin
-Wiñarquymantawan miryukunamantawan 
+﻿Wiñarquymantawan miryukunamantawan
 2009wata  hatun willaykuriy
 Wiñarquymantawan miryukunamantawan 
 Miryukunaqa sinchi allinmi runakunaq wiñananpaqqa: yanapanmanmi democratización nisqataqa, kallpachanmanmi huñukusqa runakuna rurakuyta hinallataq runachakusqap rinanpaqpis ñanchantaq, llapan tiqsimuyupi, kawsanap tiyayninpipis,  hinallataq sapa p'unchaw yachay hap’iqaypipis. 
